examples/migration_guide/bet_disk.py

```diff
@@ -0,0 +1,168 @@
+"""
+This module helps the users to use their old json files 
+containing BETDisks in the new version of Flow360.
+"""
+
+import json
+from typing import Optional
+
+import pydantic as pd
+
+from flow360.component.simulation.models.volume_models import BETDisk
+from flow360.component.simulation.primitives import Cylinder
+from flow360.component.simulation.unit_system import AngularVelocityType, LengthType, u
+
+
+def bet_disk_convert(
+    file: str,
+    save: Optional[bool] = pd.Field(False),
+    length_unit: LengthType = u.m,
+    omega_unit: AngularVelocityType = u.deg / u.s,
+):
+    """
+    Convert old BETDisks input from a json file into the new BETDisk format.
+
+    This function provides the updated version of BETDisk
+    parameters used in describing simulation cases.
+
+    Parameters
+    ----------
+    file : str
+        Name of the json file containing BETDisk information. Default is None.
+    save_to_file : bool, optional
+        Choose whether to save the output to a file/files.
+    length_unit : LengthType, optional
+        BETDisk parameters length unit. Default is u.m
+    omega_unit : AngularVelocityType, optional
+        BETDisk omega's unit. Default is u.deg / u.s.
+
+    Returns
+    -------
+    BETDisks, Cylinders
+        BETDisks defined using the provided file.
+        Cylinder defined using the provided file.
+
+    Raises
+    ------
+    AttributeError
+        If the input unit values don't exit in the 'unyt' module.
+    TypeError
+        If required file name is not provided.
+    FileNotFoundError
+        If the provided file does not exist.
+
+    Examples
+    --------
+    Example usage:
+
+    >>> BETDisks = BETDisk_convert(
+    ...     file="xv15_bet_line_hover_good.json",
+    ...     length_unit = u.ft,
+    ...     save = True,
+    ... )
+    >>> print(json.dumps(BETDisks[0].model_dump(), indent=4))
```

Comment:
this one is outdated I assume?

Reply:
Yes, thank you
the name of the function has changed, I will correct it